integrations-and-supported-tools/pytorch/notebooks/Neptune_PyTorch_Support.ipynb

```diff
@@ -0,0 +1,521 @@
+        "* [Find your project name](https://docs.neptune.ai/getting-started/installation#setting-the-project-name)\n",
+        "\n",
+        "Pass your credentials to project and api_token arguments of neptune.init()\n",
+        "\n",
+        "`run = neptune.init(api_token='<YOUR_API_TOKEN>', project='<YOUR_WORKSPACE/YOUR_PROJECT>')` # pass your credentials\n"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "YIAK4NasfQ_f"
+      },
+      "source": [
+        "run = neptune.init(project = 'common/pytorch-integration', tags='Colab Notebook', api_token='ANONYMOUS')"
+      ],
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "a4StPQOmmh_d"
+      },
+      "source": [
+        "Running this cell creates a Run in Neptune, and you can log model building metadata to it.\n",
+        "\n",
+        "**Click on the link above to open the Run in Neptune UI.** For now, it is empty, but you should keep the tab open to see what happens next"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "8CTl-ZOO1Cwr"
+      },
+      "source": [
+        "**Helper functions**"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "McylSSnLM6y5"
+      },
+      "source": [
+        "def save_model(model, name ='model.txt'):\n",
+        "    print(f'Saving model arch as {name}.txt')\n",
+        "    with open(f'{name}_arch.txt', 'w') as f:  f.write(str(model))\n",
+        "    print(f'Saving model weights as {name}.pth')\n",
+        "    torch.save(model.state_dict(), f'./{name}.pth')\n",
+        "\n",
+        "def get_obj_name(obj):\n",
+        "    return type(obj).__name__"
+      ],
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "-WbPjLDk1GcC"
+      },
+      "source": [
+        "## Step 2: Log config and hyperparameters"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "BVJKNQxIYPAC"
+      },
+      "source": [
+        "### Log Hyperparameters"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "k_8vefGCNBIJ"
+      },
+      "source": [
+        "parameters = {'lr': 1e-2,\n",
+        "              'bs': 128,\n",
+        "              'input_sz': 32 * 32 * 3,\n",
+        "              'n_classes': 10,\n",
+        "              'epochs': 50,\n",
+        "              'model_filename': 'basemodel',\n",
+        "              'device': torch.device(\"cuda:0\" if torch.cuda.is_available() else \"cpu\")\n",
+        "              }"
+      ],
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "A0Ig4exNSMwE"
+      },
+      "source": [
+        "# Log hyperameters\n",
```

Comment:
I'd drop code comments from the notebook as we have room in the markdown to do it here and we should use that